src/main/java/org/jabref/gui/mergeentries/FetchAndMergeEntry.java

```diff
@@ -76,7 +76,7 @@ public void fetchAndMerge(BibEntry entry, List<Field> fields) {
                                   })
                                   .onFailure(exception -> {
                                       LOGGER.error("Error while fetching bibliographic information", exception);
-                                      String localMessage = exception.getCause().getLocalizedMessage();
+                                      String localMessage = exception.getLocalizedMessage();
```

Comment:
Why do you use localized messge for checking here?

Reply:
I think I need to change the dialog title and content according to the localized message.

Reply:
But as far as I can see, you only rely on this variable for the check logic. The value is never actually used for the dialog content.